test/helpers.bash

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+set -u
+
+# Work around skip() in old BATS versions not handling substitutions properly
+# wrt. 'set -u'. (fixed in commit 9b2b659e5ba769752c0d00dadb3ad9b58f59cbaa).
+# This can be removed once all CI nodes have been setup with new enough BATS.
+BATS_TEARDOWN_STARTED=${BATS_TEARDOWN_STARTED:-}
```

Comment:
```suggestion
: "${BATS_TEARDOWN_STARTED:=}"
```

Reply:
I don't mind updating it like that per se (although I have to say that writing it like would not come to me naturally). However, wouldn't that change be inconsistent with the rest of the `test/*.ba{sh,ts}` code which AFAICT currently uses the `:-`-notation everywhere to set up default/fallback values ?